server/src/storage/mod.rs

```diff
@@ -21,7 +21,7 @@ mod compressed_data_manager;
 mod segment;
 mod time_series;
 mod uncompressed_data_manager;
-mod data_transfer;
+pub(crate) mod data_transfer;
```

Comment:
Could this be `pub(super)` if a method is added to `StorageEngine`?

Reply:
Fixed.